### DISCUSSION TITLE
option to set an episode "unaired"

### DISCUSSION THREAD

**Discussion by xbette** [pymedusa/Medusa#10770] [Ideas & Feature requests]
Would be sweet if you could add a option to set an episode as unaired, cause sometimes medusa trys to find episodes that isn't aired yet

As you can see here on the picture it says it have snatched it but nothing is downloaded, and no episode 2 is to find anywhere

![20-06-2022 20-04-56](https://user-images.githubusercontent.com/3235438/174657442-338b8e8c-fa06-4f82-8af4-17f7a018d24e.png)

**Comment by p0psicles** [comment]
Hmm yeah that's weird. You don't have an airdate offset configured for the show?

**Reply by xbette** [reply]
Nope not for any shows. as you see it first airs in 5 days oO

**Comment by p0psicles** [comment]
Could you zip all your logs and pm me on discord. I'll try to look tomorrow

**Reply by xbette** [reply]
Sure, I’ll fix it when I get home from work.

**Comment by p0psicles** [comment]
What does the snatch history show for this ep?

**Reply by xbette** [reply]
Where do i see that ?
